docs/Dockerfile

```diff
@@ -15,9 +15,5 @@ COPY . /docs/
 # Added arg here, so build step is invalidated each time
 ARG BUILD_NUMBER=unknown
 RUN mkdir -p /docs/_static \
- && make html
 
-# Static serving runtime
-FROM nginx
-RUN mkdir -p /usr/share/nginx/html/v1/docs/
-COPY --from=builder /docs/build/html/ /usr/share/nginx/html/v1/docs/
+CMD ["make", "livehtml"]
```

Comment:
Sorry, maar dit moeten we anders oplossen. Je gaat nu de live container ook anders laten werken.
Lokaal kan je ook zonder docker werken en dan `make livehtml` draaien.

Reply:
hmm ok. had ik niet verwacht... Ik veronderstelde dat de Dockerfile in de repository alleen gebruikt werd voor lokale setups